kubejs/client_scripts/constants.js

```diff
@@ -172,6 +172,8 @@ var regexHide = [
     /minecraft:\w+_ore/,
     /immersiveengineering:plate_/,
     /immersiveengineering:stick_/,
+    /immersiveengineering:storage_/,

```

Comment:
This should not be hidden, as the blocks can be used for building and are obtainable.

Reply:
Very very very minor thing, I'll nuke it myself :)

Reply:
but they can only be obtained via the stonecutter, so it works like every other block/ore like that? personally I would hide and maybe put an info tab/tooltip telling you those blocks are avaiable

Reply:
Hmm, good point.